vagrant_provision.sh

```diff
@@ -54,6 +56,14 @@ cp /home/vagrant/hadoop3/etc/hadoop/core-site.xml /home/vagrant/hadoop2/etc/hado
 cp /home/vagrant/hadoop3/etc/hadoop/hdfs-site.xml /home/vagrant/hadoop2/etc/hadoop/hdfs-site.xml
 
 
+# Setup Intel Storage Acceleration Library (ISA-L)
```

Comment:
If these are required for compile, you'll need to add a script to `.ci/install` and invoke it in `.travis.yml` in `before_script`. See the existing files for examples.

How long does it take to download + compile the ISAL library? We should try to reduce build time impact, or at least mitigate impact with a build cache.

Reply:
It's only about 20 mb so not that long, I think?